auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue

```diff
@@ -20,7 +20,7 @@
                   <span data-test="txt-selected-access-type">{{ selectedAccessType === AccessType.GOVN ? 'Government agency (other than BC provincial)' : 'Regular Access'}}</span>
                 </div>
                 <div
-                  v-if="canChangeAccessType"
+                  v-if="canChangeAccessType && selectedAccessType !== AccessType.GOVN"
```

Comment:
Is this condition to enable the Change button? If so, shouldn't this check for GOVM too ?

Reply:
The parent component only filters the following access types to be shown
![image](https://user-images.githubusercontent.com/78374865/148578815-78e1e844-013e-402d-ad6c-e2082fe5cec3.png)

Reply:
Refactored the code accordingly